packages/kokoas-e2e/cypress/e2e/contractPreview2/input.cy.ts

```diff
@@ -0,0 +1,204 @@
+import { beforeEach, context, cy, describe, it } from 'local-cypress';
+import { correctInputData } from './testData';
+import format from 'date-fns/format';
+import addMonths from 'date-fns/addMonths';
+
+describe(
+  '入力挙動', 
+  {
+    scrollBehavior: 'center',
+  }, 
+  () => {
+    const projId = '5a5e6cae-bea3-48e9-b679-3dcbbcc7fc60';
+    const {
+      totalContractAmt,
+      totalProfit,
+    } = correctInputData();
+  
+    beforeEach(() => {
+      cy.login();
+      cy.visit(`/project/contract/preview/v2?projId=${projId}`);
+
+    });
+
+    it('計算が合っていること', () => {
+    
+      cy.getTextInputsByLabel('契約合計金額')
+        .type(totalContractAmt.toString())
+        .blur();
+
+      cy.getTextInputsByLabel('粗利額')
+        .type(totalProfit.toString())
+        .blur();
+
+      cy.log('計算されていることを確認する');
+      cy.contains('span', '税率')
+        .siblings('span')
+        .invoke('text')
+        .then((textTaxRate) => {
+        // getNumber from text
+          const taxRate = 1 + (Number(textTaxRate.replace(/[^0-9]/g, '')) / 100);
+          return cy.wrap(taxRate);
+        })
+        .then((taxRate) => cy
+          .contains('span', '税抜金額')
+          .siblings('span')
+          .invoke('text')
+          .then((textTaxExcludeAmt) => {
+            const taxExcludedAmt = Number(textTaxExcludeAmt.replace(/[^0-9]/g, ''));
+            return cy.wrap(taxExcludedAmt)
+              .should('eq', Math.round(totalContractAmt / taxRate));
+          }))
+        .then((taxExcludedAmt) => cy
+          .contains('span', '原価')
+          .siblings('span')
+          .invoke('text')
+          .then((textCost) => {
+            const cost = Number(textCost.replace(/[^0-9]/g, ''));
+            cy.wrap(cost)
+              .should('eq', Math.round(taxExcludedAmt - totalProfit));
+            return cy.wrap({
+              taxExcludedAmt,
+              cost,
+            });
+          }))
+        .then(({
+          taxExcludedAmt,
+          cost,
+        }) => cy
+          .contains('span', '粗利率')
+          .siblings('span')
+          .invoke('text')
+          .then((textProfitRate) => {
+          // get profitRate with decimal point
+            const profitRate = Number(textProfitRate.replace(/[^0-9.]/g, ''));
+            console.log(taxExcludedAmt, cost, profitRate);
+         
+            /** 利益率 = ( 単価 - 原価) / 単価 */
+            const calculatedTaxRate = ((taxExcludedAmt - cost) /  taxExcludedAmt) * 100;
+        
+            cy.wrap(profitRate.toFixed(2))
+              .should('eq', calculatedTaxRate.toFixed(2));
+          }));
+    });
+
+    context(
+      '支払いの挙動', 
+      {
+        testIsolation: false,
+      }, 
+      () => {
+        const dividedAmt = Math.round(totalContractAmt / 4);
+        const payments = [
+          ['契約金', 'contract'],
+          ['着手金', 'initial'],
+          ['中間金', 'interim'],
+          ['最終金', 'final'],
+        ];
+
+        it('契約金すること', () => {
```

Comment:
```suggestion
        it('契約金額にコンマが追加されるのを確認する', () => {
```

修正前の内容だと、何をするテストか分からないように感じました。
修正案を書きましたが、必ずしも上の通りでなくて大丈夫です。